sdk/mixedreality/Azure.MixedReality.RemoteRendering/src/Azure.MixedReality.RemoteRendering.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyTitle>Microsoft Azure Mixed Reality ARR Client</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Azure MixedReality</PackageTags>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Authors>Microsoft</Authors>
+    <PackageProjectUrl>https://azure.microsoft.com/services/remote-rendering/</PackageProjectUrl>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <Nullable>enable</Nullable>
```

Comment:
Please don't enable nullable annotation for public APIs we currently ship Azure SDKs without nullable annotations.

Reply:
It seems that lots of libraries use this. I tried removing it, but unfortunately, I'm pulling in shared-source files from the auth library and they won't compile without it.

Reply:
@pakrym Can you elaborate on this? There a ton of other libraries in here using nullable and we've had .NET customers asking for nullable annotations so that they can have those insights in their C#8 enabled projects. I'm not sure I understand this requirement.

Reply:
Among the Track 2 libraries only Azure.Core uses nullable annotations in public APIs. Which libraries have you seen are using annotated APIs?

Currently, we don't have the tooling to track breaking changes in nullability annotations, and not enough customer ask to invest in the infrastructure just yet.

Reply:
I have removed <Nullable>nullable from my lib and added #nullable to the shared source file where it was needed.

For the record, these project files switch it on:

sdk\communication\Azure.Communication.Administration\src\Azure.Communication.Administration.csproj:
sdk\communication\Azure.Communication.Chat\src\Azure.Communication.Chat.csproj:
sdk\communication\Azure.Communication.PhoneNumbers\src\Azure.Communication.PhoneNumbers.csproj:
sdk\communication\Azure.Communication.Sms\src\Azure.Communication.Sms.csproj:
sdk\core\Azure.Core\src\Azure.Core.csproj:
sdk\core\Azure.Core.Amqp\src\Azure.Core.Amqp.csproj:
sdk\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj:
sdk\core\Microsoft.Azure.Core.NewtonsoftJson\src\Microsoft.Azure.Core.NewtonsoftJson.csproj:
sdk\core\System.Memory.Data\src\System.Memory.Data.csproj:
sdk\keyvault\samples\keyvaultproxy\src\AzureSamples.Security.KeyVault.Proxy.csproj:
sdk\mixedreality\Azure.MixedReality.Authentication\src\Azure.MixedReality.Authentication.csproj:

Reply:
We've talked to the communication team this week, they would be removing nullability from public APIs. The `core` libraries having them is expected.